Makefile

```
@@ -1,67 +1,195 @@
+# no manual `go build` or `go test` invocations happen here except for
+# the `lint` target which runs the linter via `go run` (unchanged behaviour).
+#
+# Override the Bazel binary if needed:
+#   make BAZEL=bazelisk test
+#   make BAZEL=/usr/local/bin/bazel build
+#
+# VERSION is injected by Bazel through --workspace_status_command=scripts/bazel-stamp.sh
+# which emits STABLE_KTN_VERSION; no -ldflags are needed in Make.
 
+.PHONY: help build install fmt \
+        test test-unit test-validate test-race test-bench \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)